src/_guides/language/language-tour.md

```diff
@@ -4553,11 +4553,11 @@ class Llama {
 }
 ```
 
-In the generated documentation, `[Food]` becomes a link to the API docs
-for the Food class.
+In the generated documentation, `[food]` becomes a link to the API docs
+for the `Food` class.
```

Comment:
This is no longer true. The text `[food]` becomes a link to the parameter. The user can still navigate to the class from the type annotation on the parameter declaration, but not from the dartdoc reference.

Reply:
@bwilkerson I tried to add some class documentation to cover the desired results, what do you think about these latest changes?

Reply:
Those changes look fine to me, but I'll leave it to @kwalrath to decide whether they add to the quality of the documentation.